TestVectors/Ed25519Signature2020/proofConfigEdSig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "Ed25519Signature2020",
+  "created": "2023-02-24T23:36:38Z",
+  "verificationMethod": "https://vc.example/issuers/5678#z6MkrJVnaZkeFzdQyMZu1cgjg7k1pZZ6pvBQ7XJPt4swbTQ2",
+  "proofPurpose": "assertionMethod",
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
```

Comment:
```suggestion
```

Reply:
https://www.w3.org/ns/credentials/examples/v2 exists now and thus this change request is no longer valid.